PyInstaller/hooks/hook-osgeo.py

```diff
@@ -0,0 +1,76 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2015, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+
+from PyInstaller.utils.hooks import collect_data_files
+from PyInstaller.compat import is_win, is_darwin
+
+import os
+import sys
+
+# The osgeo libraries require auxiliary data and may have hidden dependencies.
+# There are several possible configurations on how these libraries can be
+# deployed.
+# This hook evaluates the cases when:
+# - the `data` folder is present "in-source" (sharing the same namespace folder
+#   as the code libraries)
+# - the `data` folder is present "out-source" (for instance, on Anaconda for
+#   Windows, in PYTHONHOME/Library/data)
+# In this latter case, the hook also checks for the presence of `proj` library
+# (e.g., on Windows in PYTHONHOME) for being added to the bundle.
+#
+# This hook has been tested with gdal (v.1.11.2 and 1.11.3) on:
+# - Win7 64bit
+# - Ubuntu 15.04 64bit
+# - Mac OS X Yosemite 10.10
+#
+# TODO: Fix for gdal>=2.0: 'NameError: global name 'help' is not defined'
+
+# flag used to identify an Anaconda environment
+is_conda = False
+
+# Auxiliary data:
+#
+# - general case (data in 'osgeo/data/gdal'):
+datas = collect_data_files('osgeo', subdir='data/gdal')
```

Comment:
Please use `os.path.join`. Using `/` as directory separator is not platform-independent.